### DISCUSSION TITLE
Pushing image to new GitHub Container Repository Doesn't Create a Package?

### DISCUSSION THREAD

**Discussion by dschinkel** [community/community#26021] [Actions]
<p>I assumed this would create a new package just like the older Github Docker Repository Pushes would create.  I see that I pushed successfully but do not see a new package gened?</p>
<p>Is that expected behavior and if so where do I find the image after the fact if I want to query it or see it somewhere?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/658fefb740262f3de98148ab75965c03fb2caa61.jpeg" data-download-href="/uploads/short-url/eusBCzsRZYGaZzwtAOpIEpIwTRf.jpeg?dl=1" title="Screen Shot 2020-09-03 at 2.35.15 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/338798/181094797-5930afe8-30d2-4809-8b10-e4abebffa937.jpeg" alt="Screen Shot 2020-09-03 at 2.35.15 AM" data-base62-sha1="eusBCzsRZYGaZzwtAOpIEpIwTRf" width="690" height="301" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/658fefb740262f3de98148ab75965c03fb2caa61_2_690x301.jpeg, https://aws1.discourse-cdn.com/github/original/2X/6/658fefb740262f3de98148ab75965c03fb2caa61.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/6/658fefb740262f3de98148ab75965c03fb2caa61.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/658fefb740262f3de98148ab75965c03fb2caa61_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-09-03 at 2.35.15 AM</span><span class="informations">800×350 107 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>this still shows my old Docker Package when I was using GitHub Docker Repository<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6b0786c9df2a87ce541e45c0196f06d940902576.jpeg" data-download-href="/uploads/short-url/fgPdjNyfKV8bp5NiBUad3AS3kBE.jpeg?dl=1" title="Screen Shot 2020-09-03 at 2.38.00 AM copy" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/338798/181094798-1b30f6e3-f4ac-4a1b-b497-e605d15e3c91.jpeg" alt="Screen Shot 2020-09-03 at 2.38.00 AM copy" data-base62-sha1="fgPdjNyfKV8bp5NiBUad3AS3kBE" width="690" height="204" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/6/6b0786c9df2a87ce541e45c0196f06d940902576_2_690x204.jpeg, https://aws1.discourse-cdn.com/github/optimized/2X/6/6b0786c9df2a87ce541e45c0196f06d940902576_2_1035x306.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/6/6b0786c9df2a87ce541e45c0196f06d940902576_2_1380x408.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6b0786c9df2a87ce541e45c0196f06d940902576_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-09-03 at 2.38.00 AM copy</span><span class="informations">2114×628 212 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/dschinkel">@dschinkel</a> ,</p>
<p>I have an example workflow includes two jobs (see <a href="https://github.com/PackagesRML/TestDocker/blob/128c0ebaa3aa60ebc038240cc5e53788b4e71eb1/.github/workflows/CI.yml" rel="nofollow noopener">here</a>):</p>
<ul>
<li>
<strong>publish_to_GPDR:</strong> Build and publish imgae to GitHub Packages Docker Registry (GPDR).</li>
<li>
<strong>publish_to_GHCR:</strong> Build and publish imgae to GitHub Container Registry (GHCR).</li>
</ul>
<p>The two jobs are separate from each other and run in parallel.</p>
<p>If you want to only push the image to GitHub Container Registry, you can ignore <strong>publish_to_GPDR</strong>, and in <strong>publish_to_GHCR</strong>, you do not need to firstly build and push the image to GitHub Packages Docker Registry. You can directly build and push the image GitHub Container Registry. Just like you can see the all steps in my workflow.</p>

**Comment by dschinkel** [comment]
<p>That’s what I did.  My question here is it’s not creating a Package as a result.  I did the same thing you did in your GHCR example already.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/dschinkel">@dschinkel</a> ,</p>
<p>You can navigate to “<a href="https://github.com/orgs/" rel="nofollow noopener">https://github.com/orgs/</a>&lt;your_owner_name&gt;/packages” to view the Docker package you push to GitHub Container Registry.</p>

**Comment by dschinkel** [comment]
<p>I don’t have an org though…just an owner.  I assumed it would just show up here just like the old GitHub Docker Repository would push it to:</p>
<p><a href="https://github.com/dschinkel/we-do-tdd/packages" class="onebox" target="_blank" rel="nofollow noopener">https://github.com/dschinkel/we-do-tdd/packages</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b130341f8e136ea7a64f22973c264528540c83ce.jpeg" data-download-href="/uploads/short-url/phtMR9fzn6ELq8tNbXtG7wfgxxs.jpeg?dl=1" title="Screen Shot 2020-09-03 at 2.54.58 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/338798/181094803-997ac922-e004-4bcf-bf78-a9665ad3f19e.jpeg" alt="Screen Shot 2020-09-03 at 2.54.58 AM" data-base62-sha1="phtMR9fzn6ELq8tNbXtG7wfgxxs" width="690" height="358" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b130341f8e136ea7a64f22973c264528540c83ce_2_690x358.jpeg, https://aws1.discourse-cdn.com/github/original/2X/b/b130341f8e136ea7a64f22973c264528540c83ce.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/b130341f8e136ea7a64f22973c264528540c83ce.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b130341f8e136ea7a64f22973c264528540c83ce_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-09-03 at 2.54.58 AM</span><span class="informations">1000×519 110 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
<div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/ced13e9f8dde309e22cecaa1d8091a4c42beb73e.jpeg" data-download-href="/uploads/short-url/tvAF8ibEPpge8TOXrT0sgqbn99Q.jpeg?dl=1" title="Screen Shot 2020-09-03 at 2.58.10 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/338798/181094806-b6405d97-f324-4179-bd0c-63eb27aab831.jpeg" alt="Screen Shot 2020-09-03 at 2.58.10 AM" data-base62-sha1="tvAF8ibEPpge8TOXrT0sgqbn99Q" width="690" height="195" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/ced13e9f8dde309e22cecaa1d8091a4c42beb73e_2_690x195.jpeg, https://aws1.discourse-cdn.com/github/original/2X/c/ced13e9f8dde309e22cecaa1d8091a4c42beb73e.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/c/ced13e9f8dde309e22cecaa1d8091a4c42beb73e.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/ced13e9f8dde309e22cecaa1d8091a4c42beb73e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-09-03 at 2.58.10 AM</span><span class="informations">1000×283 77.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The docs for the new container repository are so lacking.  They really should have updated the documentation better before releasing the Beta Github Container Repository IMO.  There is a lot of inferred info or context that the GitHub team knows but we don’t still left out in this process.  They didn’t completely think about the lowest common denominator in that section of the GitHub Action docs.</p>
<p>So I’m left here with “Ok it pushed it successfully it says, but <em>where</em> the hell is it?”</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/dschinkel">@dschinkel</a> ,</p>
<p>If the Docker package is published to your user account’s GitHub Container Registry, you follow the below steps to view the package:<br>
<strong>Your profile</strong> &gt; <strong>Packages</strong><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b2b67852be0718261ef4e33783cdf6ee18020fc9.gif" data-download-href="/uploads/short-url/puXVEGsq1i3cJBeZfFK1X4U7biV.gif?dl=1" title="example02" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181094808-553a081d-e061-4fd7-81a5-08b5129e9f2b.gif" alt="example02" data-base62-sha1="puXVEGsq1i3cJBeZfFK1X4U7biV" width="690" height="354" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b2b67852be0718261ef4e33783cdf6ee18020fc9_2_690x354.gif, https://aws1.discourse-cdn.com/github/optimized/2X/b/b2b67852be0718261ef4e33783cdf6ee18020fc9_2_1035x531.gif 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/b/b2b67852be0718261ef4e33783cdf6ee18020fc9_2_1380x708.gif 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b2b67852be0718261ef4e33783cdf6ee18020fc9_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">example02</span><span class="informations">1567×804 162 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If the Docker package is published to an organization’s GitHub Container Registry, you follow the below steps to view the package:<br>
<strong>Your organizations</strong> &gt; click the organization you want to access &gt; <strong>Packages</strong><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/f/fa5e2e7272e7cf1ed8525c3206c0fb7ae9e7ac9c.gif" data-download-href="/uploads/short-url/zIR9k9pg1CHF5Prge2wAiNZDtlq.gif?dl=1" title="example" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181094809-c78ce78f-b348-4a3e-b23b-b363184f94a4.gif" alt="example" data-base62-sha1="zIR9k9pg1CHF5Prge2wAiNZDtlq" width="690" height="354" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/f/fa5e2e7272e7cf1ed8525c3206c0fb7ae9e7ac9c_2_690x354.gif, https://aws1.discourse-cdn.com/github/optimized/2X/f/fa5e2e7272e7cf1ed8525c3206c0fb7ae9e7ac9c_2_1035x531.gif 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/f/fa5e2e7272e7cf1ed8525c3206c0fb7ae9e7ac9c_2_1380x708.gif 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/f/fa5e2e7272e7cf1ed8525c3206c0fb7ae9e7ac9c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">example</span><span class="informations">1567×804 276 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by dschinkel** [comment]
<p>:rofl: OMG seriously?  I had no idea there was a global Packages tab that looks just like the packages tab in a repo.  Is that new?</p>

**Comment by dschinkel** [comment]
<p>Seriously <strong>GitHub Team please update your Container Repository docs</strong>!  We’ve always mainly looked at a repo’s Packages tab, not global account.</p>

**Comment by dschinkel** [comment]
<p>3am, I’m spent, this is crazy.  Time to go to bed<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e78e952e752614492893aa339412bec6712f4b7f.jpeg" data-download-href="/uploads/short-url/x2rMdO5s3hQf6ghEfSVObJy2XP9.jpeg?dl=1" title="Screen Shot 2020-09-03 at 3.11.36 AM" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/338798/181094815-cc99b218-e106-4cc8-87e1-b5c0a2a098c9.jpeg" alt="Screen Shot 2020-09-03 at 3.11.36 AM" data-base62-sha1="x2rMdO5s3hQf6ghEfSVObJy2XP9" width="690" height="411" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/e/e78e952e752614492893aa339412bec6712f4b7f_2_690x411.jpeg, https://aws1.discourse-cdn.com/github/original/2X/e/e78e952e752614492893aa339412bec6712f4b7f.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/e/e78e952e752614492893aa339412bec6712f4b7f.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e78e952e752614492893aa339412bec6712f4b7f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-09-03 at 3.11.36 AM</span><span class="informations">1000×597 150 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Thanks again for the 2nd pair of eyes @ <a href="https://github.community/u/brightran">brightran</a></p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/dschinkel">@dschinkel</a> ,</p>
<p>The docs should have mentioned where the docker images store when you publish the package to GitHub Container Registry. (see <a href="https://docs.github.com/en/packages/getting-started-with-github-container-registry/about-github-container-registry" rel="nofollow noopener">here</a>)</p>
<blockquote>
<p>Store container images within your organization and user account, rather than a repository.</p>
</blockquote>

**Comment by dschinkel** [comment]
<p>yea it’s not obvious at all, they really needed to launch that beta better…that sucks.</p>

**Comment by dschinkel** [comment]
<blockquote>
<p>Store container images within your organization and user account, rather than a repository</p>
</blockquote>
<p>This means nothing to me as far as the actual GitHub UI.  This doesn’t infer to most people “hey go to your main profile to look for the Container images!”…I mean lets be real.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/dschinkel">@dschinkel</a> ,</p>
<p>Okay, I agree that the description in the docs is too short and omit some helpful details.</p>

**Comment by dschinkel** [comment]
<p>This is starting to remind me of Azure.  I sure hope GitHub doesn’t create 100 places in the UI to find things like you have to hunt and do in Azure.  Usability is key.</p>

**Comment by BrightRan** [comment]
<p><a class="mention" href="/dschinkel">@dschinkel</a> ,</p>
<p>I recommend that you can directly report your feedback about packages <a href="https://support.github.com/contact/feedback?contact%5Bcategory%5D=packages" rel="nofollow noopener">here</a>. That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>
